PSGSuite/Public/Calendar/Get-GSCalendarSubscription.ps1

```diff
@@ -30,7 +36,13 @@ function Get-GSCalendarSubscription {
         $User = $Script:PSGSuite.AdminEmail,
         [parameter(Mandatory = $false,Position = 1)]
         [String[]]
-        $CalendarId
+        $CalendarId,
+        [Parameter(Mandatory = $false, Position = 2)]
+        [bool]
+        $ShowDeleted = $false,
```

Comment:
To keep things in line with the same patterns elsewhere (and general common patterns for PowerShell functions in general), can we swap both `$ShowDeleted` and `$ShowHidden` to switch parameters with no default value (as they default to `$false` already)?

Reply:
Made the updates needed and pushed to your fork's master! Getting this merged asap.